ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh

```
@@ -63,13 +63,19 @@ download_brew_rpms() {
     fi
 
     # Download the latest RPMs from brew: latest release (ec, rc and zstream), nightly, Y-1 zstream and Y-2 zstream
-    y_version="$(cut -d'.' -f2 "${src_path}/Makefile.version.$(uname -m).var")"
-    bash -x ./test/bin/manage_brew_rpms.sh download "4.${y_version}" "${out_path}" "ec" || echo "WARNING: Failed to download EC RPMs for 4.${y_version}"
-    bash -x ./test/bin/manage_brew_rpms.sh download "4.${y_version}" "${out_path}" "rc" || echo "WARNING: Failed to download RC RPMs for 4.${y_version}"
-    bash -x ./test/bin/manage_brew_rpms.sh download "4.${y_version}" "${out_path}" "zstream" || echo "WARNING: Failed to download zstream RPMs for 4.${y_version}"
-    bash -x ./test/bin/manage_brew_rpms.sh download "4.${y_version}" "${out_path}" "nightly" || echo "WARNING: Failed to download nightly RPMs for 4.${y_version}"
-    bash -x ./test/bin/manage_brew_rpms.sh download "4.$((${y_version} - 1))" "${out_path}" "zstream" || echo "WARNING: Failed to download zstream RPMs for 4.$((${y_version} - 1))"
-    bash -x ./test/bin/manage_brew_rpms.sh download "4.$((${y_version} - 2))" "${out_path}" "zstream" || ( echo "WARNING: Failed to download zstream RPMs for 4.$((${y_version} - 2))" && return 1 )
+    # Source common_versions.sh for version variables that handle cross-major boundaries (e.g. 5.0 -> Y-1=4.22, Y-2=4.21)
+    export UNAME_M="$(uname -m)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)